build.gradle.kts

```diff
@@ -46,14 +44,6 @@ tasks.withType<Detekt>().configureEach {
     }
 }
 
-tasks.withType<DependencyUpdatesTask> {
-    rejectVersionIf {
-        candidate.version.isNonStable()
-    }
-}
-
-fun String.isNonStable() = "^[0-9,.v-]+(-r)?$".toRegex().matches(this).not()
-
```

Comment:
After #31, we can remove this plugin?

Reply:
I would rather keep this plugin. It allows having an overview of newly published versions without relying on Github entirely.

Reply:
[Using a Gradle init script](https://github.com/ben-manes/gradle-versions-plugin#using-a-gradle-init-script) might be a better solution?

Reply:
Mmmm not really. How would you use it?

Reply:
You can put [this file](https://github.com/Goooler/DemoApp/blob/trunk/gradle/init.d/dependencyUpdates.gradle) under you device's `~/.gradle/init.d`, and just run `./gradlew dependencyUpdates` in any project you need to cheking.

Reply:
It would be outside the template then.

Reply:
97123b5d9b3d16e449734967c612412e63655d7c